static/ankersrv.css

```diff
@@ -1,41 +1,79 @@
 .nav-link,
 .btn-link {
-  color: #88f387;
-  text-decoration: none;
+    color: #88f387;
+    text-decoration: none;
 }
 
 .nav-link.active,
 .btn-primary {
-  color: #000000 !important;
-  background-color: #88f387 !important;
-  border: #41a03f;
+    color: #000000 !important;
+    background-color: #88f387 !important;
+    border: #41a03f;
 }
 
 .nav-link:hover,
 .btn:hover {
-  color: #000000;
-  background-color: #adf3ac;
-  border-color: #41a03f;
+    color: #000000;
+    background-color: #adf3ac;
+    border-color: #41a03f;
 }
 
 a.text-light {
-  text-decoration: none;
+    text-decoration: none;
 }
 
 a.text-light:hover {
-  color: #88f387 !important;
+    color: #88f387 !important;
 }
 
 pre,
 code {
-  color: #88f387;
-  background-color: #292929;
+    color: #88f387;
+    background-color: #292929;
 }
 
 .form-label {
-  font-weight: bolder;
+    font-weight: bolder;
 }
 
 code#loginFilePath {
-  font-weight: lighter;
+    font-weight: lighter;
+}
+
+.print-status {
+    font-size: medium;
+}
+
+.progress {
+    height: 40px;
+}
+
+.icon {
+    background-size: cover;
+    display: inline-block;
+    position: fixed;
```

Comment:
Why are we using `position: fixed`, if we are not specifying the position?

Reply:
This prevents the icon from tiling. If someone like @just-trey has a better way to do this, I'm open to hear it :)

Reply:
You are correct this is the easiest way to keep an image from trying to tile

Reply:
why not use
background-repeat: no-repeat

Reply:
@chrivers, this is fixed with my push. also, you will see new clock icons added are custom-built by me. What I found online was too fat.

Reply:
Looks great, thanks.